hart.adoc

```diff
@@ -0,0 +1,10 @@
+[[hart]]
+== Hart Requirements
+The BRS specification is minimally prescriptive on the RISC-V hart requirements. It is anticipated that detailed requirements will be driven by target market segment and product/solution requirements. The SEE mandates the minimal set required for a compliant system implementation of the BRS.
+
+The BRS requires harts to least be compliant to:
+
+* RVA20S64, as BRS governs the interactions between 64-bit OS supervisor-mode software and 64-bit firmware, when employed.
```

Comment:
Can we cite the UEFI section on RISC-V calling convention as a source of additional RISC-V hart requirements that needs to be referenced by an implementation?

Reply:
This seems confusing and unnecessary.  The BRS depends on the UEFI.  The BRS should not restate UEFI requirements.

Reply:
Not restate. Just point to the section that is of interest. This is what the ACPI section does. The UEFI spec is large enough that referring to the section of interest makes sense.

Reply:
Which section in UEFI spec? 2.3.7 RISC-V Platforms, but that was before your changes, Andrei. I'll put that there for the time being.

Reply:
May not be the best place, but I added a note about how to find UEFI expectations related to this.